firmware/brake/kia_soul_ev_niro/src/brake_control.cpp

```diff
@@ -200,7 +211,27 @@ static void read_brake_pedal_position_sensor(
     brake_pedal_position_s * const value )
 {
     cli();
-    value->high = analogRead( PIN_BRAKE_PEDAL_POSITION_SENSOR_HIGH );
-    value->low = analogRead( PIN_BRAKE_PEDAL_POSITION_SENSOR_LOW );
+    value->A = analogRead( PIN_BRAKE_PEDAL_POSITION_SENSOR_A );
+    value->B = analogRead( PIN_BRAKE_PEDAL_POSITION_SENSOR_B );
     sei();
 }
+
+uint8_t check_brake_pedal_position_data(
```

Comment:
/cc @rebpdx are these changes going to conflict with the override/fault detection optimization that you've done?